onlineweb4/settings/django.py

```diff
@@ -23,6 +23,13 @@
 # might be 100% safe to remove, but keep it safe
 ATMOIC_REQUESTS = True
 
+CACHES = {
+    "default": {
+        "BACKEND": "django.core.cache.backends.db.DatabaseCache",
```

Comment:
If you use the DatabaseCache I don't think this actually speeds anything up?

Reply:
Some db queries may be extra slow so it's worth caching, no? 👍

Reply:
I see what you mean though. But this requires no further infra, easy to implement and it may have some gains at least

Reply:
Yes, but this is caching it _in the database_ making it probably almost as slow as running the query itself, you can make a performance-test with locust. I have [scaffolding](https://github.com/dotkom/onlineweb4/commit/fa06eb27577eeb7feb6c64d6870b5abb93bee691) from a previous test

Reply:
Hmm so you're saying that there are no queries so slow that they're worth caching in the same db. You may be right, I haven't done extensive testing. I thought it would be worth it for queries with a lot of joins but the gains may be negligible compared to the network delay. 

This is only a draft btw

Reply:
There are definitively queries worth caching / not performing, and we _should_ have caching like this in most places.

Thanks to sentry we actually have quite a bit of insight into this, so we do actually know what causes the most issues: https://dotkom.sentry.io/profiling/?project=204971&statsPeriod=90d

Essentially it's mostly the event-api